### DISCUSSION TITLE
How to download temperature data from CESM2-LE

### DISCUSSION THREAD

**Discussion by robbiemallett** [NCAR/cesm2-le-aws#14] [Q&A]
Sorry if this is the wrong place to ask, but I'm trying to download hourly or 3-hourly surface temperature data from CESM2-LE.

I've tried here:
https://www.earthsystemgrid.org/dataset/ucar.cgd.cesm2le.atm.proc.3hourly_ave.html

But it seems that only a bunch of quite random variables are hosted (volume mixing ratios). Is there somewhere where we can get more standard variables?

I've tried using this notebook, but am unable to get intake-ESM to install compatibly with X-Array.

https://ncar.github.io/cesm2-le-aws/kay_et_al_lens2.html

**Comment by danielmwatkins** [comment]
Hi Robbie! I'm not part of the CESM2-LE team but I have been using the notebook that you linked. What system are you using? On my Mac, I've been able to get intake installed but it took some work. I had to make a new Conda environment, and then install things one at a time. I installed packages in the following order:
`proplot`, `jupyter`, `geocat-comp`, `intake`,  `intake-esm`, `aiohttp`, `s3fs`.
The first two are for plotting and for notebooks, I installed those first since they have more limited requirements than some of the others. The last three aren't listed in the notebook, but are required to make `intake-esm` work.

However, that may be a moot point. 6-hourly data is available for some periods from CESM1-LE, but the temperature data on AWS for CESM2-LE is limited to daily and monthly frequencies as far as I can tell.
